cpg_workflows/workflow.py

```diff
@@ -929,9 +929,9 @@ def __init__(
                 'Workflow already initialised. Use get_workflow() to get the instance'
             )
 
-        self.dry_run = dry_run or get_config()['workflow'].get('dry_run')
+        self.dry_run = dry_run or get_config(True)['workflow'].get('dry_run')
 
-        analysis_dataset = get_config()['workflow']['dataset']
+        analysis_dataset = get_config(True)['workflow']['dataset']
```

Comment:
Won't it still print it twice for non-dry-runs?

Reply:
Oh… only does the printing if `_config` actually got consed up.

:shrug: Okay, or could be clearer if the `self.dry_run` line was moved below the other line's block of four lines. Or really we could have `Workflow.__init__` start with `config = get_config(True)` and use that instead of calling `get_config()` a zillion times…

Reply:
Yeah, moving the dry run would have made sense, I should have done that. I tried banking the get_config call into a dummy variable first but the linter didn't like that.

Reply:
The get_config calls are a bit weird looking, but I prefer the consistency compared to pulling it into an object and having to pass the object around.

Reply:
No worries. Geez linter 🙃